app/javascript/components/party/party-manifestos/index.js

```diff
@@ -0,0 +1,142 @@
+        return fetch(`/parties/${party_acronym}/manifesto.json`)
+            .then(res => res.json())
+            .catch((error) => {
+                console.log(error);
+            });
+    }
+
+    getParty() {
+      const { party_acronym } = this.props.match.params;
+
+      // party_acronym is already encoded
+      return fetch(`/parties/${party_acronym}.json`)
+          .then(res => res.json())
+          .catch((error) => {
+              console.log(error);
+          });
+    }
+
+    componentDidMount() {
+        const itemsPromise = this.getItems()
+        const manifestoDataPromise = this.getManifestoData()
+        const partyPromise = this.getParty()
+
+        Promise.all([itemsPromise, manifestoDataPromise, partyPromise]).then((results) => {
+            const [section, manifesto, party] = results;
+
+            this.setState({
+                title: manifesto.title,
+                sections: manifesto.sections,
+                section: section,
+                party: party
+            });
+        });
+    }
+
+    getSelectedKey(sections, section_id) {
+        return section_id ? [section_id] : []
+    }
+
+    getOpenKey(sections, section_id) {
+        let openKey = []
+        sections.forEach((section) => {
+            if (section.subsections.length > 0) {
+                section.subsections.forEach((subsection) => {
+                    if (subsection.id.toString() === section_id) {
+                        openKey = [section.id.toString()]
+                    }
+                })
+            }
+        })
+
+        return openKey.length ? openKey : [];
+    }
+
+    render() {
+        const { sections, title, section, party } = this.state;
+        const { section_id, party_acronym } = this.props.match.params;
+
+        return (
+            <Layout className="party-manifesto">
+                <LayoutHeader />
+                <Layout.Content>
+                    <PartyHeader party={party} subtitle={party_acronym + " - Programa"} showManifestoLink={false} />
+                    <Layout>
+                        <Sider width={200} className="party-manifesto-sider">
+                            {sections.length && (
+                                <ManifestoSider
+                                    sections={sections}
+                                    party_acronym={party_acronym}
+                                    section_id={section_id}
+                                    selectedKey={this.getSelectedKey(sections, section_id)}
+                                    openKey={this.getOpenKey(sections, section_id)}
+                                />
+                            )}
+                        </Sider>
+                        <Layout.Content>
+                            <ManifestoSection title={title} section={section} section_id={section_id} />
+                        </Layout.Content>
+                    </Layout>
+                    <LayoutFooter />
```

Comment:
@nancruz @jagoncalves14 I think this should be below `</Layout.Content>` but only works here..

Reply:
Fixed while being inside <Layout>